README.md

```diff
@@ -188,24 +188,23 @@ use DocusignBundle\EnvelopeBuilder;
 use DocusignBundle\Events\PreSignEvent;
 use DocusignBundle\Events\DocumentSignatureCompletedEvent;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Symfony\Component\HttpKernel\Event\ExceptionEvent;
-use Symfony\Component\HttpKernel\KernelEvents;
 
 class PreSignSubscriber implements EventSubscriberInterface
 {
     public static function getSubscribedEvents()
     {
         // return the subscribed events, their methods and priorities
         return [
-            PreSignEvent::class => 'preSign'
-            DocumentSignatureCompletedEvent::class => 'onDocumentSignatureCompleted'
+            PreSignEvent::EVENT_NAME => 'preSign',
```

Comment:
Using this constant was in fact just a convenient way to name it.
In the v4.5, i went for the classname. Still working and more alike to the 4.* of EventDispatcher.